bears/js/JSONFormatBear.py

```diff
@@ -10,10 +10,7 @@
 
 
 class JSONFormatBear(LocalBear):
-    try:
-        DecodeError = json.decoder.JSONDecodeError
-    except AttributeError:
-        DecodeError = ValueError
+    DecodeError = json.decoder.JSONDecodeError
```

Comment:
Yep, I saw this in the AppVeyor build, but it was already 3:30am. How can I
test that `except` branch then?

On Tue, Sep 6, 2016, 9:50 AM AbdealiJK <notifications@github.com> wrote:

> In bears/js/JSONFormatBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/767#discussion_r77587012>
> :
>
> > @@ -10,10 +10,7 @@
> >
> >
> >  class JSONFormatBear(LocalBear):
> > -    try:
> > -        DecodeError = json.decoder.JSONDecodeError
> > -    except AttributeError:
> > -        DecodeError = ValueError
> > +    DecodeError = json.decoder.JSONDecodeError
>
> It does not ahttps://
> docs.python.org/3/library/json.html#json.JSONDecodeError -
> JSONDecodeError is new in 3.5
> Hence it wont exist in 3.4 and 3.3. Which is why this is there.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/767/files/e88fbd755e29e863c155cdbcce509d0304ac35a4..eb3c64c7fefacdd4cbac7eee6786f97374bc085b#r77587012>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABAuVJpWYjXOOrXrvNHYC23ZfbOa1as_ks5qnRswgaJpZM4JzMuc>
> .
>